libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/WPImageGetter.java

```diff
@@ -20,7 +20,12 @@
 /**
  * ImageGetter for Html.fromHtml()
  * adapted from existing ImageGetter code in NoteCommentFragment
+ *
+ *
+ * @deprecated Use org.wordpress.android.util.image.getters.WPCustomImageGetter instead.
```

Comment:
It looks like there is only one other place we use this class, should we just update that as well, if not in this PR, maybe in the next one? Maybe you were already planning to do so, so ignore this comment if that's the case :)

Reply:
;) I was planning to update it in another PR and remove the WPImageGetter entirely.